config/foreman_maintain.yml.example

```diff
@@ -0,0 +1,10 @@
+# Directory where the logs are stored.
+# The default file is named as /log/foreman_maintain.log
+:log_dir: 'log'
```

Comment:
@iNecas ,
No any specific reason. 
Generally, user can only allow to provide directory for logs not a log file. 

Assuming that if user is frequently changing directory path of log files due to his environment setup then at least we should know the log file name. Using which it will become easy to locate and find directory path.

After allowing users to set log file, it will create difficulties for us during troubleshooting.
In the future, what If these logs are included in foreman-debug then file name will work.

Please let me know your opinion.